src/table/table.scss

```diff
@@ -16,75 +15,205 @@
     box-sizing: border-box;
   }
 
-  > .iui-tables-body {
-    width: 100%;
-    @include themed {
-      background-color: t(iui-color-background-1);
-    }
+  .iui-table-header {
+    @include iui-table-header;
+  }
+
+  .iui-table-body {
+    @include iui-table-body;
+  }
+
+  .iui-cell {
+    @include iui-table-cell;
   }
 
-  &.iui-zebra-striping > .iui-tables-body > .iui-tables-row:nth-child(even) {
+  &.iui-zebra-striping > .iui-table-body > .iui-row:nth-child(even):not(.iui-selected) {
     @include themed {
       background: rgba(t(iui-color-foreground-body-rgb), 0.02);
     }
   }
 
-  &.iui-tables-fixed > .iui-tables-body {
-    max-height: 300px;
+  &.iui-fixed > .iui-table-body {
+    max-height: $iui-baseline * 27;
     overflow-x: hidden;
     overflow-y: scroll;
     overflow: overlay;
   }
+}
 
-  .iui-select .iui-select-button {
-    background-color: transparent;
-    border: none;
+@mixin iui-table-header {
+  width: 100%;
```

Comment:
Lots of places width: 100% is put.
Was it needed? Cause tried to play around with it in header, did not change anything, I think...

Reply:
I was able to remove all instances of `width: 100%;` from the SCSS without any visible issues.